CliClient/tests/markdownUtils.js

```diff
@@ -66,4 +67,41 @@ describe('markdownUtils', function() {
 		}
 	}));
 
+	it('should remove Markdown syntax elements from the text', asyncTest(async () => {
+		const inputStrings = [
+			'', // Empty string
+			'This is some plain text', // Plain text
+			'## This is a header', // Header syntax
+			'This is a text with **bold** and *italicized* text', // Text with annotations
+			'This is a text with __bold__ and _italicized_ text', // Text with annotations alternate form
+			'[link to google](https://www.google.com/)', // Link
+			'> This is a blockquote\n And another line', // Blockquote
+			'* List item\n* List item', // Unordered list
+			'- List item\n- List item', // Unordered list
+			'1. List item\n2. List item', // Ordered list
+			'This is some `inline code`', // Inlined code
+		];
+
+		const expectedOutputStrings = [
+			'',
+			'This is some plain text',
+			'This is a header',
+			'This is a text with bold and italicized text',
+			'This is a text with bold and italicized text',
+			'link to google',
+			'This is a blockquote\n And another line',
+			'List item\nList item',
+			'List item\nList item',
+			'List item\nList item',
+			'This is some inline code',
+		];
+
+		assert(inputStrings.length ==  expectedOutputStrings.length, 'inputStrings array and expectedOutputStrings array don\'t have the same size');
```

Comment:
Could you change this to `expect(inputStrings.length).toBe(expectedOutputStrings.length)` and remove the dependency to `assert`? Please also make sure to update package.json and package-lock.json. Other than that, the PR looks good so we can merge.

Reply:
One query, if there is a dependency in the `package.json` of the project root, does it automatically install in the sub-folders as well, like for the TerminalClient and the ReactNativeClient?

Reply:
Removed the assert dependency.

Reply:
> 

One query, if there is a dependency in the package.json of the project root, does it automatically install in the sub-folders as well, like for the TerminalClient and the ReactNativeClient?

No it doesn't, the dependencies at the root are only for tooling, eslint, etc.

Reply:
> Removed the assert dependency.

I mean it should be removed from here too and replace by `expect(inputStrings.length).toBe(expectedOutputStrings.length)`